packages/firebase_auth/firebase_auth/example/web/index.html

```diff
@@ -22,6 +22,9 @@
     };
     // Initialize Firebase
     firebase.initializeApp(firebaseConfig);
+
+    // Configure web auth for emulator
+    //firebase.auth().useEmulator('http://localhost:9099');
```

Comment:
Is this required when the main.dart for the example already has it?

![image](https://user-images.githubusercontent.com/5347038/105691326-02171e00-5ef5-11eb-84d7-03c572b485a7.png)

Reply:
Good point, I think that should be deleted. Will fix and re-push.

Reply:
fixed, pushed